### ISSUE TITLE
plotting vs stimulus set sweep number

### ISSUE THREAD

**Issue by timjarsky** [AllenInstitute/MIES#302]
It's common for users to acquire multiple repeats of the same stimulus set. To plot Y data against stimset sweep number the user can currently do something like what is shown below for a 5 sweep stimulus set acquired 3X.

```
max(data([0,1000], channels(AD), sweeps())

vs

[0;1;2;3;4;0;1;2;3;4;0;1;2;3;4;]

```

The above does not work when there are different stimulus sets on different headstages or when user skips sweeps. So, it would be nice to have something like: listFromStimSetSweepNumber() that returns the `set cycle count` for each sweep (on each channel).

**Comment by t-b** [comment]
@ukos-git https://alleninstitute.github.io/MIES/labnotebook-docs.html

**Comment by t-b** [comment]
Closed via #345.
